test/e2e_node/image_list.go

```diff
@@ -151,27 +152,39 @@ func PrePullAllImages() error {
 	}
 	images := framework.ImageWhiteList.List()
 	klog.V(4).Infof("Pre-pulling images with %s %+v", puller.Name(), images)
+
+	var wg sync.WaitGroup
+	var once sync.Once
 	for _, image := range images {
-		var (
-			err    error
-			output []byte
-		)
-		for i := 0; i < maxImagePullRetries; i++ {
-			if i > 0 {
-				time.Sleep(imagePullRetryDelay)
+		wg.Add(1)
```

Comment:
could you instead do 
```go
wg.Add(len(images))
```
outside the loop as the length is pre-determined?

Reply:
done, PTAL